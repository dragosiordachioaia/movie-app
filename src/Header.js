import React, { Component } from "react";

export default class Header extends Component {
  constructor(props) {
    super(props);

    // this.displayStuff = this.displayStuff.bind(this);
  }

  displayStuff() {}

  render() {
    return <div> Header </div>;
  }
}
