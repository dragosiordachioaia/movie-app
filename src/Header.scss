@import "variables";

.header {
  background-color: $color-secondary;
  color: #fff;
  height: $header-height;
  display: flex;
  flex-direction: row;
  margin-top: 0;

  .logo-container {
    text-align: center;
    box-sizing: border-box;
    width: $sidebar-width;
    position: relative;
    background-color: $color-secondary-alt;

    img {
      height: 1.3em;
      top: 50%;
      left: 50%;
      position: absolute;
      transform: translate(-50%, -50%);
    }
  }
}
