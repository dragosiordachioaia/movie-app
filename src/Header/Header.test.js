import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import Header from "./Header";

jest.mock("SearchBar/SearchBar", () => "SearchBar");

let component;
let instance;

beforeEach(() => {
  component = renderer.create(<Header />);
  instance = component.getInstance();
});

it("renders without crashing", () => {
  instance.props = {};
  const result = instance.render();
  expect(result).toMatchSnapshot();
});
