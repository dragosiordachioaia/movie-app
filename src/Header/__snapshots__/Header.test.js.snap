// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="header"
>
  <div
    className="logo-container"
  >
    <img
      alt="logo"
      src="./logo.png"
    />
  </div>
  <div
    className="search-bar-container"
  >
    <SearchBar />
  </div>
</div>
`;
